lib/scrape.js

```diff
@@ -165,6 +166,45 @@ async function findOrganization({
   return null
 }
 
+async function findWiki(name) {
+  const APIurl = wdk.searchEntities({
+    search: name,
+    format: 'json',
+  })
+  const search = await fetch(APIurl)
+  const response = await search.json()
+  if (response && response.search.length) {
+    const wikidataurl = response.search[0].url
+    const wikidataid = response.search[0].id
+
+    const entityAPIurl = wdk.getEntities({
+      ids: [wikidataid],
+      props: ['sitelinks'],
+      format: 'json',
+    })
+    const sitelinks = await fetch(entityAPIurl)
+    let sitelink = await sitelinks.json()
+    sitelink = sitelink.entities[wikidataid].sitelinks
+    if (sitelink && sitelink['enwiki']) {
```

Comment:
'enwiki' is choosing language (and repeated below); the site will soon be i18n's, so this needs to be in a variable which can be altered easily.

But why filter here?  This is scraping. More data is better than none.  Push all the extracted data through to the next layer, normalising and improving data. (e.g. `enwiki->` -> `wikipedia->en->` ?)

Then the UI templates choose which sitelinks they want to display.